### PR TITLE
Orb updated with suggestions

### PR DESCRIPTION
**Pull request by raphdev**
This PR implements changes discussed in [#39 of Salus](https://github.com/coinbase/salus/issues/39#issuecomment-489435490).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +9,10 @@ A CircleCI Orb for Salus
 | salus_executor | CircleCI executor to use that specifies Salus environment | `coinbase/salus:2.4.1` | See [executor reference](https://circleci.com/docs/2.0/configuration-reference/#executors-requires-version-21)|
```
Comment (raphdev) [inline README.md]: We should update this to reflect that we now use `coinbase/salus:latest` as default:
```suggestion
 | salus_executor | CircleCI executor to use that specifies Salus environment | `coinbase/salus:latest` | See [executor reference](https://circleci.com/docs/2.0/configuration-reference/#executors-requires-version-21)|
```

File: orb.yml [hunk 2]
```diff
@@ -51,14 +54,28 @@ jobs:
...
+          working_directory: /home
 
       - run: 
           name: Run scan
           command: |
-            SALUS_CONFIGURATION="file://salus-configuration.yaml" /usr/local/bundle/bin/bundle exec ./bin/salus scan
+            configuration_file="<<parameters.configuration_file>>"
+            if [[ -n "repo/${configuration_file}" ]]; then
+              if [[ ! -f "repo/${configuration_file}" ]]; then
+                echo "$repo/{configuration_file} does not exist, aborting."
```
Comment (raphdev) [inline orb.yml]: I like that we're being descriptive and failing to indicate we can't run as configured.

**Review by raphdev — COMMENTED** [review]
This looks good to me. Nice job!

File: orb.yml (hunk 2 above)
Comment (jsulinski) [reply orb.yml]: Fixed a few typos here.

File: README.md (hunk 1 above)
Comment (jsulinski) [reply README.md]: I think pegging to a known working version is preferred over the latest tag, this change wasn't intentional so I reverted it (and updated to 2.4.2).

File: orb.yml [hunk 3]
```diff
@@ -26,7 +26,7 @@ jobs:
       report_uri:
         description: Defines where to send Salus reports
         type: string
-        default: "file://test/salus-report.json"
+        default: "file://../salus-report.json"
```
Comment (jsulinski) [inline orb.yml:29]: This is what I meant about using relative paths.

File: orb.yml [hunk 4]
```diff
@@ -51,14 +54,27 @@ jobs:
...
+            configuration_file="<<parameters.configuration_file>>"
+            if [[ -n "${configuration_file}" ]]; then
+              if [[ ! -f "repo/${configuration_file}" ]]; then
+                echo "repo/${configuration_file} does not exist, aborting."
+                exit 1
+              else
+                bundle exec /home/bin/salus scan --repo_path $CIRCLE_WORKING_DIRECTORY --config "file://${configuration_file}"
+              fi
+            else
+              bundle exec /home/bin/salus scan --repo_path $CIRCLE_WORKING_DIRECTORY --config "file://../salus-configuration.yaml"
```
Comment (jsulinski) [inline orb.yml:71]: Another relative path.

**Comment by jsulinski** [comment]
Thanks so much for the suggestions and review! I think this is good to go.

File: README.md (hunk 1 above)
Comment (raphdev) [reply README.md]: This sounds good to me. Is the intention that the latest production orb version follows the latest stable Salus version? If so, this means we have to follow versioning closely and potentially make releases following any version update. I'm not against this, as long as we're clear on what to expect as default. What do you think makes sense?

**Comment by raphdev** [comment]
No problem! Feel free to make a release whenever you're ready. Also, I'm happy to help do some more of the legwork, just let me know what you need.

File: README.md (hunk 1 above)
Comment (jsulinski) [reply README.md]: Yeah, we'll have to follow versioning closely and release new ones when Salus does. I'm happy to help maintain this as I'll be closely following Salus' development, and if it becomes a burden we can shift to latest tag. 

When I said preferred, I was referencing this comment: https://github.com/coinbase/salus/pull/41#issuecomment-487819029

**Comment by jsulinski** [comment]
Okay, thanks! Will push it today. I don't think there is much legwork left. Any suggestions on improving the README? It's been a pleasure working with you, any qualms with adding a Contributors section with thanks?

**Pull request merged**